src/main/java/uk/gov/hmcts/befta/TestAutomationConfig.java

```diff
@@ -48,6 +49,22 @@ public UserTokenProviderConfig getUserTokenProviderConfig() {
         return UserTokenProviderConfig.DEFAULT_INSTANCE;
     }
 
+    public Ordering getDefaultCollectionAssertionMode() {
+        Ordering returnValue = Ordering.ORDERED;
+
+        try {
+            Ordering defaultCollectionAssertionMode =
+                    Ordering.of(EnvironmentVariableUtils.getOptionalVariable("DEFAULT_COLLECTION_ASSERTION_MODE"));
+            if (defaultCollectionAssertionMode != null) {
+                returnValue = defaultCollectionAssertionMode;
+            }
+        } catch (IllegalArgumentException iae) {
+            returnValue = Ordering.ORDERED;
```

Comment:
no need for this.

Reply:
But good to generate a warning log.

Reply:
Agreed, but better than an empty catch block - Need to work out logging levels in befta as not everything needs to be logged to console by default